tests/test_directory_sync.py

```diff
@@ -174,3 +174,10 @@ def test_list_directories(self, mock_directories, mock_request_method):
         response = self.directory_sync.list_directories()
         assert response.status_code == 200
         assert response.response_dict == mock_directories
+
+    def test_delete_directory(self, mock_directories, mock_request_method):
+        mock_response = Response()
+        mock_response.status_code = 200
```

Comment:
We can. Would you want this updated to 202 throughout the entire SDK?

Reply:
It'll be just for the delete directory since that's the status code the API returns. 🙂 We have mocked 202 in our Node SDK [here](https://github.com/workos-inc/workos-node/blob/master/src/directory-sync/directory-sync.spec.ts#L31) as an example.

Reply:
Complete